pkg/config/config.go

```diff
@@ -62,6 +62,9 @@ const (
 	PacWatcherName    = "pipelines-as-code-watcher"
 	PacWebhookName    = "pipelines-as-code-webhook"
 
+	// Path to a chains public key
+	PublicKeyPath = "testdata/chains/key"
```

Comment:
I didn't mean to move this line to a different file :) Only to create similar `const` section in original file